extension/data_loader/file_data_loader.cpp

```diff
@@ -17,7 +17,16 @@
 #include <fcntl.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#ifndef _WIN32
 #include <unistd.h>
+#else
+#include <io.h>
+#endif
+
+#ifdef _WIN32
+#include <stddef.h>
+using ssize_t = ptrdiff_t;
+#endif
```

Comment:
Is it possible to move this part to `runtime/platform/compiler.h`?

Reply:
Sure.

Reply:
Updated.